packages/pass-style/src/passStyle-helpers.js

```diff
@@ -165,7 +166,7 @@ const makeCheckTagRecord = checkProto => {
       (isObject(tagRecord) ||
         (!!check &&
           CX(check)`A non-object cannot be a tagRecord: ${tagRecord}`)) &&
-      (isFrozen(tagRecord) ||
+      (isNoTrapping(tagRecord) ||
         (!!check && CX(check)`A tagRecord must be frozen: ${tagRecord}`)) &&
```

Comment:
Changing this to

```suggestion
        (!!check && CX(check)`A tagRecord must be no-trapping: ${tagRecord}`)) &&
```

is technically more correct. But will be understood much less often.